specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/applications.json

```diff
@@ -219,10 +219,61 @@
           },
           "202": {
             "description": "Accepted response definition."
+          },
+          "204": {
+            "description": "No Content response definition."
           }
         },
         "x-ms-long-running-operation": true
```

Comment:
Can you also specify the x-ms-long-running-operation-options here?
Please refer doc http://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options

Reply:
> Can you also specify the x-ms-long-running-operation-options here?
> Please refer doc http://azure.github.io/autorest/extensions/#x-ms-long-running-operation-options

Hi @RamyasreeChakka  Thanks for your comment, I have fixed this.